sdk/communication/azure-communication-chat/azure/communication/chat/_models.py

```diff
@@ -56,6 +55,57 @@ def _to_generated(self):
             share_history_time=self.share_history_time
         )
 
+class ChatAttachment(object):
```

Comment:
```suggestion
class ChatAttachment:
```

Reply:
was following ChatParticipant, will also make the change for ChatParticipant as well